packages/types/surveys.ts

```diff
@@ -41,8 +41,17 @@ export const ZSurveyProductOverwrites = z.object({
   darkOverlay: z.boolean().nullish(),
 });
 
+export const ZSurveyBackground = z.object({
+  bg: z.string().nullish(),
+  bgType: z.string().nullish(),
+  brightness: z.number().nullish(),
+  // darkOverlay: z.boolean().nullish(),
```

Comment:
Can we remove this comment, if not required?

Reply:
Sure, i'll remove that  :)